boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TheaterInDerInnenstadtCollector.kt

```diff
@@ -18,9 +18,11 @@ class TheaterInDerInnenstadtCollector : TwoStepEventCollector<Element>("theateri
     private val eventUrl = baseUrl + "spielplan/"
 
     override fun getAllUnparsedEvents(): List<Element> {
-        return Jsoup
-            .parse(fetcher.fetchUrl(eventUrl))
-            .select("div.event")
+        val document = Jsoup.parse(fetcher.fetchUrl(eventUrl))
+        val events = document.select("div.event")
+        val logo = document.selectFirst("a.tm-logo img")
+        events.forEach { it.append(logo.toString()) }
```

Comment:
honestly i would prefer to pass this as a pair, with the Element and the Logo as two different objects